Assignment3.txt

```diff
@@ -0,0 +1,9 @@
+Albert Zhang
+1)Not sure if this is right please correct me if I'm wrong.
+	1. O(n) = n
+	2. O(n) = n + n^2
```

Comment:
Close! since it's big O notation, you don't have to care about the lower terms, so this just becomes O(n^2).